### ISSUE TITLE
Can't Arrow Up into empty Columns Block

### ISSUE THREAD

**Issue by jeryj** [WordPress/gutenberg#51554]
The column block won't allow an arrow up keypress to navigate into an empty column block.

Debugging this, it seems like the arrow key navigation code is all working correctly. It's correctly identifying the appender button as the one to focus, but it's unable to focus it.

### Reproduction
- Add a column block
- Add a paragraph or any block after it
- Try to arrow up from the paragraph block into the column block

### Video
https://github.com/WordPress/gutenberg/assets/967608/d63b64de-e58d-4fde-a3ed-1cb5b18eaa62

**Comment by andrewserong** [comment]
Just linking a potentially similar writing flow issue: #55177

**Comment by ellatrix** [comment]
It's not a writing flow issues, but a CSS issues due to the use of visibility: none. Fixed in #55197.
